### PR TITLE
David.benque/signal priority

### PR DESCRIPTION
**Pull request by dbenque**
Ability to define priority for signals.

The syntax for the `node-conditions` parameters has changed, for example to pass a delay now:
`--node-conditions='cluster-upgrade={"delay":"6h"}'`

To add a priority:
`--node-conditions='cluster-upgrade={"priority":10}'`

For the priority, the higher the value, the higher the priority. A node with signal priority 5 will be taken before anode with a signal of priority 0. Negative values are possible. Default value is .

### CODE REVIEW

File: internal/candidate_runner/sorters/node_annotations.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			}
+		}
+
+	}
+	if n2.Labels != nil {
+		if str, ok := n2.Labels[NodeAnnotationDrainASAPKey]; ok {
+			a2 = 2
+			if str != "" {
+				if a2, err = strconv.Atoi(str); err != nil {
+					a2 = 1 // go back to default value in case of conversion error
```
Comment (dhenkel92) [inline internal/candidate_runner/sorters/node_annotations.go:31]: The default of a2 was 2 above

**Review by dhenkel92 — APPROVED** [review]
Perfect! 🙂

File: internal/candidate_runner/sorters/node_annotations.go (hunk 1 above)
Comment (dbenque) [reply internal/candidate_runner/sorters/node_annotations.go:31]: 🤦 , good catch I will fix this.

**Review by dhenkel92 — APPROVED** [review]

File: internal/cli/commands.go [hunk 2]
```diff
@@ -68,6 +87,18 @@ func ReadFromURL(url string) ([]byte, error) {
 	return b, nil
 }
 
+func (h *CLICommands) cmdGroupGraphLast() error {
+	params := url.Values{}
+	params.Add("group-name", h.groupName)
```
Comment (dhenkel92) [inline internal/cli/commands.go:92]: We should check if the group name was set and show a nice error message if not.

**Review by dhenkel92 — APPROVED** [review]

**Review by dhenkel92 — APPROVED** [review]

**Pull request merged**